packages/terra-arrange/src/Arrange.module.scss

```diff
@@ -37,3 +37,16 @@
     align-self: flex-start;
   }
 }
+
+.fit-block {
+  .icon-wrapper {
```

Comment:
Is this class intended for terra-arrange ? there are no reference in Arrange component for class `icon-wrapper` !!

Reply:
This class has been added in section header component, i have moved these styles into section header module scss file.